### ISSUE TITLE
Some questions about how the code works

### ISSUE THREAD

**Issue by Brainxwq** [deep-bci/ERBCPSI#2]
Do I need to download the data of two data sets first，Then use MATLAB to process the data？

**Comment by deep-bci** [comment]
> Do I need to download the data of two data sets first，Then use MATLAB to process the data？

Yes. You need first download the EEG dataset, and then compute the functional connections among EEG electrods using PSI, PLV, etc. Then you can apply the proposed code to these functional connectivity matrices for emotion recognition.

**Comment by Brainxwq** [comment]
Thank for your prompt reply.

**Comment by Brainxwq** [comment]
May I have your QQ number(or other contact way)? I still have some questions. Thank you very much!

**Comment by Brainxwq** [comment]
Or can you share the address of the paper?Thanks a lot

**Comment by deep-bci** [comment]
> Or can you share the address of the paper?Thanks a lot

This work has just been accepted by ICCSIP 2020 (International Conference on Cognitive Systems and Information Processing). The final paper can be available when it is included in the proceedings.

**Comment by Brainxwq** [comment]
OK，thanks




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "deep-bci"<notifications@github.com&gt;; 
发送时间: 2020年12月14日(星期一) 晚上9:23
收件人: "deep-bci/ERBCPSI"<ERBCPSI@noreply.github.com&gt;; 
抄送: "464129748"<464129748@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [deep-bci/ERBCPSI] Some questions about how the code works (#2)





  
Or can you share the address of the paper?Thanks a lot
  
This work has just been accepted by ICCSIP 2020 (International Conference on Cognitive Systems and Information Processing). The final paper can be available when it is included in the proceedings.

**Comment by Brainxwq** [comment]
Hello, I am not very clear about the process of MATLAB.I was wondering if I could run the program directly with the sample data you gave in the file.

**Comment by deep-bci** [comment]
> Hello, I am not very clear about the process of MATLAB.I was wondering if I could run the program directly with the sample data you gave in the file.

Do you know the basic process of training a neural network?

**Comment by Brainxwq** [comment]
Yeah




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "deep-bci"<notifications@github.com&gt;; 
发送时间: 2020年12月14日(星期一) 晚上10:26
收件人: "deep-bci/ERBCPSI"<ERBCPSI@noreply.github.com&gt;; 
抄送: "464129748"<464129748@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [deep-bci/ERBCPSI] Some questions about how the code works (#2)





  
Hello, I am not very clear about the process of MATLAB.I was wondering if I could run the program directly with the sample data you gave in the file.
  
Do you know the basic process of training a neural network?

**Comment by Qunliyau** [comment]
can you share the address of the paper? Thanks！

**Comment by deep-bci** [comment]
> 

https://link.springer.com/chapter/10.1007/978-981-16-2336-3_3

**Comment by Qunliyau** [comment]
> > 
> 
> https://link.springer.com/chapter/10.1007/978-981-16-2336-3_3



> Thank you so much for your reply.
